src/components/styles/ArtworkDetails.module.css

```diff
@@ -0,0 +1,34 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+.mainContainer {
+  padding: 5vh;
+  background-color: #fd5294;
+}
+
+.mainContainer h4 {
+  padding: 0 0 3vh 0;
+  font-size: 25px;
+}
+
+.detailsContainer {
+  display: flex;
+}
+
+.detailsContainer p {
+  padding: 1.5vh 0;
+}
+
+.leftDetails {
+  width: 50%;
+}
+
+.rigthDetails a {
+  color: black;
+}
+
+.bold {
+  font-weight: bold;
+}
```

Comment:
- [ ] Nice work :clap: I can see it's an interesting topic with good specs, however, the requirement says that you should follow the template design(colors, layout,...) kindly make sure your project has the same layout as in the original design.

your design | Original design 
----|---
![image](https://user-images.githubusercontent.com/65068771/189510748-17449d2c-4851-4656-9b11-b91bec54d952.png) ![image](https://user-images.githubusercontent.com/65068771/189510861-f8f22b25-7169-4ed8-9258-037d3377aebc.png) | ![image](https://user-images.githubusercontent.com/65068771/189510899-9126d1f6-2893-4ed3-81a8-4efe862397ea.png) ![image](https://user-images.githubusercontent.com/65068771/189510912-edf7e8f3-f8ad-4d82-b6d6-c02f8b00ce59.png)